connection.go

```diff
@@ -1072,7 +1080,7 @@ func (conn *Connection) putFuture(fut *Future, req Request, streamId uint64) {
 		if fut = conn.fetchFuture(reqid); fut != nil {
 			resp := &Response{
 				RequestId: reqid,
-				Code:      OkCode,
+				Code:      CodeOk,
```

Comment:
Shouldn't we add this one to the migration guide?

Reply:
I reverted this change, the field can contain an error value too - it may be not a good idea to make a separate type.